app/services/analytics_events.rb

```diff
@@ -4635,6 +4635,14 @@ def reactivate_account_visit
     track_event('Reactivate Account Visited')
   end
 
+  # A previous reCAPTCHA assessment was annotated
+  # @param [String] assessment_id Assessment reference identifier
+  # @param [String] reason Reason for annotating the assessment
+  # @param [String] annotation Type of annotation
```

Comment:
```suggestion
  # @param ["INITIATED_TWO_FACTOR","PASSED_TWO_FACTOR"] reason Reason for annotating the assessment
  # @param ["LEGITIMATE","FRAUDULENT"] annotation Type of annotation
```

Reply:
I've only implemented a handful we're currently using, but technically there could be as many as [15 reasons](https://cloud.google.com/recaptcha-enterprise/docs/reference/rest/v1/projects.assessments/annotate#reason) and [5 annotations](https://cloud.google.com/recaptcha-enterprise/docs/reference/rest/v1/projects.assessments/annotate#annotation), unsure how scalable that is to repeat them out here? Would be nice if we could somehow reference an external type similar to what can be done with exporting / importing types in TypeScript.

Maybe I'll just add the specific strings and we can sort out later if/when we add more.

Reply:
Indirectly addressed through dcc31d13d9, though maybe `Hash` could be a more specific type in the YARDoc with how it's embedded in existing events.

Reply:
now this event is not logged at all (or referenced by the other events), should we remove it?

Reply:
Yep, we should. Good catch..